packages/contracts-bedrock/test/universal/Specs.t.sol

```diff
@@ -878,6 +879,20 @@ contract Specification_Test is CommonTest {
         _addSpec({ _name: "DeputyGuardianModule", _sel: _getSel("superchainConfig()") });
         _addSpec({ _name: "DeputyGuardianModule", _sel: _getSel("version()") });
 
+        // DeputyPauseModule
+        _addSpec({ _name: "DeputyPauseModule", _sel: _getSel("version()") });
+        _addSpec({ _name: "DeputyPauseModule", _sel: _getSel("foundationSafe()") });
+        _addSpec({ _name: "DeputyPauseModule", _sel: _getSel("deputyGuardianModule()") });
+        _addSpec({ _name: "DeputyPauseModule", _sel: _getSel("superchainConfig()") });
+        _addSpec({ _name: "DeputyPauseModule", _sel: _getSel("deputy()") });
+        _addSpec({ _name: "DeputyPauseModule", _sel: _getSel("usedNonces(bytes32)") });
+        _addSpec({ _name: "DeputyPauseModule", _sel: _getSel("pauseMessageTypehash()") });
+        _addSpec({ _name: "DeputyPauseModule", _sel: _getSel("deputyAuthMessageTypehash()") });
+        _addSpec({ _name: "DeputyPauseModule", _sel: _getSel("setDeputy(address,bytes)") });
+        _addSpec({ _name: "DeputyPauseModule", _sel: _getSel("setDeputyGuardianModule(address)") });
+        _addSpec({ _name: "DeputyPauseModule", _sel: _getSel("eip712Domain()") });
+        _addSpec({ _name: "DeputyPauseModule", _sel: _getSel("pause(bytes32,bytes)"), _auth: Role.PAUSEDEPUTY });
```

Comment:
Are we using `_addSpec` correctly here? `setDeputy` and `setDeputy` can only be called by the foundation safe, and `pause` can be called by anyone so should have no auth

Reply:
Unfortunately these specs are not automatically enforced by anything, which would be a pretty big lift, esp. for the `pause()` method which requires EIP712 signatures.

> pause can be called by anyone so should have no auth

IMO the `_auth` arg is correct here, even though the auth isn't implemented with a `msg.sender` check it's still enforcing that the `deputy` has signed a message.

Otherwise I agree and think this is the right diff:

```suggestion
        _addSpec({ _name: "DeputyPauseModule", _sel: _getSel("setDeputy(address,bytes)"), _auth: Role.DEPUTYGUARDIAN });
        _addSpec({ _name: "DeputyPauseModule", _sel: _getSel("setDeputyGuardianModule(address)") }, _auth: Role.DEPUTYGUARDIAN);
        _addSpec({ _name: "DeputyPauseModule", _sel: _getSel("eip712Domain()") });
        _addSpec({ _name: "DeputyPauseModule", _sel: _getSel("pause(bytes32,bytes)"), _auth: Role.PAUSEDEPUTY });
```